docs/language_concepts/00_data_types.md

```diff
@@ -197,6 +197,12 @@ fn main() {
 }
 ```
 
+You can instantiate a new array of a fixed size with the same value repeated for each element. The following example instantiates an array od length 32 where each element is of type Field and has the value 0.
```

Comment:
Typo: "an array od length" -> "an array of length"

Reply:
thanks jake, fixed here. https://github.com/noir-lang/docs/pull/293